website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -119,6 +129,30 @@ dbt sl export --saved-query sq_number1 --export-as table --alias new_export
 ```
 </details>
 
+### Exports for multiple saved queries
+
+Use the command, `dbt sl export-all`, to run exports for multiple saved queries at once. This is different to the `dbt sl export` command, which only runs exports for a singular saved query.  For example, to run exports for multiple saved queries, you can use:
+
+#### Example
+
+```bash
+dbt sl export-all
+```
+
+#### Output
```

Comment:
Same as above. We can probably just separate these with a sentence rather than headers
```suggestion
The output would look something like this: 
```